.codesandbox/workspace.json

```diff
@@ -0,0 +1,20 @@
+{
+  "responsive-preview": {
+    "Mobile": [
+      320,
+      675
+    ],
+    "Tablet": [
+      1024,
+      765
+    ],
+    "Desktop": [
+      1400,
+      800
+    ],
+    "Desktop  HD": [
+      1920,
+      1080
+    ]
+  }
+}
```

Comment:
This is a great way of making the application responsive. Inspired!

Reply:
This file is made by default by code sandbox.😅